src/new_index/schema.rs

```diff
@@ -142,6 +141,43 @@ impl ScriptStats {
     }
 }
 
+/// key: (u8, [u8;32]) tuple of token_type and payload
+/// value: ScriptStats
+#[derive(Hash, PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
+pub struct StatsKey(u8, [u8; 32]);
+
+pub type StatsMap = HashMap<StatsKey, ScriptStats>;
+
+impl StatsKey {
+    pub fn from_color_id(color_id: &Option<ColorIdentifier>) -> Self {
+        match color_id {
+            Some(c) => {
+                let payload = &serialize(c);
+                let payload = *array_ref![payload, 1, 32];
+                StatsKey(c.token_type.clone() as u8, payload)
+            }
+            None => StatsKey(0, [0; 32]),
```

Comment:
In Tapyrus Core, `TokenType#none` is defined. How about defining a similar type in rust-tapyrus?
https://github.com/chaintope/tapyrus-core/blob/master/src/coloridentifier.h#L17

Or it is better to define 0, [0; 32] as a constant.

Reply:
https://github.com/chaintope/rust-tapyrus/pull/28

Reply:
Update rust-tapyrus version(0.4.4) 
- support TPC token type.
- implement Hash trait for ColorIdentifier

And replace Hash key type from StatsKey to ColorIdeitifer